### DISCUSSION TITLE
Go to a specific table page when refreshing table

### DISCUSSION THREAD

**Discussion by salvomil** [rappasoft/laravel-livewire-tables#369] [General]
Hi, I used the component for a test in a project and I'd like to go to a specific page when I refresh a table. More specific, if there is an item in page, i.e. pag 4, and I delete the  item when I refresh the page the table go back to first page. Is it possible to refresh the page and go directly to pag 4?

Any suggestion? Maybe I don't how to do it. Thank you

**Comment by rappasoft** [comment]
Resetting back to the first page is a feature. You can try to set `$this->page = 4;` after your delete clause and see if that works.

**Reply by salvomil** [reply]
Oh!!! Immediate answer! Thank you!!!
